### PR TITLE
Buildifier and CI config cleanup

### PR DESCRIPTION
**Pull request by jin**
https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/README.md#legacy-format

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -2,11 +2,6 @@ workspace(name = "rules_jvm_external")
 
 android_sdk_repository(name = "androidsdk")
 
-local_repository(
```
Comment (dkelmer) [inline WORKSPACE]: was this unused?

File: defs.bzl [hunk 2]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 load("@rules_jvm_external//:coursier.bzl", "coursier_fetch")
-load("@rules_jvm_external//:specs.bzl", "maven", "parse", "json")
```
Comment (dkelmer) [inline defs.bzl]: was maven unused? I'm very impressed buildifier knows how to check for that

File: WORKSPACE (hunk 1 above)
Comment (jin) [reply WORKSPACE]: Yes, unused

Comment (jin) [reply WORKSPACE]: Buildifier says this name was defined earlier by the workspace() declaration

File: defs.bzl (hunk 2 above)
Comment (jin) [reply defs.bzl]: Yep! Caught by buildifier

**Pull request merged**